### ISSUE TITLE
Define CNCF-related Entities and their Relationships

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#45]
## User story
1. As a data engineer
2. I want to define the CNCF-specific key entities and their relationships
3. So that the NER model can recognize CNCF-specific information

## Acceptance criteria
* Utilise the extracted data from the CNCF landscape

* Create a list that contains:
    * Entity types: Project_Name, Technology_Name, (Organization_Name), ...
    * Entities: Kubernetes, Docker, gRPC,...
    * Relationships: Depends_On, Complements, (Conflicts_with), ...

* Here is an example:
    * **CNCF entities**
        * Project_Name: Kubernetes, ...
        * Technology_Name: Docker, gRPC, ...
        * (Organization Name: Google, Red Hat, ...)
        * ...

    * **Relationships between entities** 
        ```
        "entity1": "Docker",
        "relationship": "Complements",
        "entity2": "Kubernetes",
        "description": "Docker containers can be managed and coordinated using Kubernetes."```

* Store the list in a format that can be used for the NER model training

* As for this part of the work, it does not have to be automated

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project-specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dominic0df** [comment]
merged with the issue Annotate a Dataset for NER training
